src/test/app.test.js

```diff
@@ -0,0 +1,49 @@
+import App from '../components/app';
+import Todo from '../components/todo';
+
+const app = new App();
```

Comment:
You have really done well, great job guys.
Perhaps due to workload 🙂 , you missed testing the method `addTodo` in `app.js`
kindly add it.

I trust you guys can add this before merging 👀  on you.

Reply:
Thanks for the commitment and we assure you that we will first add that test and then we will merge the branch.